### ISSUE TITLE
Axis values is not documented

### ISSUE THREAD

**Issue by Timmmm** [vega/vega#2111]
See [`values`](https://vega.github.io/vega/docs/axes/):

> values - **Array** - Explicitly set the visible axis tick and label values.

Array of what? In Typescript it is just `any[]` which is not very helpful. What the heck do I put in there? I tried an array of strings (my scale is ordinal), but they just overlap each other at the top.

**Comment by jheer** [comment]
The `values` property takes an array of values for which each entry is a valid member of the scale domain. For ordinal or band scales, the values must exactly match those in the domain. I've updated the axis and legend documentation to state this explicitly, and here is a [basic example](https://vega.github.io/editor/#/url/vega/N4KABGBEAkDODGALApgWwIaQFxUQFzwAdYsB6UgN2QHN0A6agSz0QFcAjOxge1IRQyUa6SgFY6AK1jcAdpAA04KAHdGAExbYwAFgAMuxREgpG1fFoBM+w1ELo1axjOpbRipZDXo8mHAG0lCFAIEKgZdFRkLUgfdgAbKJtQyAp0ONZkWC0A0NDgSHhvGm4AJwBPaIBBBSgI7lYZPEsADgBfJNz8wrxi8uiAIRrIOoamnFFRdsDOgqLqUoqcSABhIZHGrW0AZinckK65heiAETXUeo2cAE4ARl29sAOe+b6lgFEzi7GwZruOvNmzyOSwAYp9Rq4dv99oDeosoABxcGXMA3K73PZPOHRAASyO+zQA7K1phAALrTEnk9xGWCmcJxLL+abBXKQcKRaJ4bjcOJ4RiEBSkqCpdJRHDADEhSCybLCoKQZBURpMqAlZDwPBYc6sWDIbhUEpDViELw9aJm1ioSBSgFK5Aq6LqzXa+p6+pNeRGE1m8VQSU24UU0JUsBkmlQBBpTJyvLC9kRP2QAAeUYSQr2MTKhCT7HQMjUGbZam4GCcWnyZt8UFi6a9kAAZoxkHFC0tutjbWr89Qk6oNIgi8k7A4nC4cLo6Lo3PGShc22A8CUMpSOqzkhyk2U04l4yWy3IJZ5vNWYuh4okoE2WwvhjqNl32Yx4H6lxloZASj2kyYzE1KUo4YgB46DJjGzJ5FApTNiikDsNwBClkMO7RKmhR1iKaQZKqfiQNU9arPWSJkmAUrAFBJQwd8kAJA2nqRuhW4oaRgERpAGAlAA1jhLLxng2ZJs6nrxg2c7WkeVbYGeF42h+DrwNwah+uubIOj0RoSvK0rJhWkAoUsaHRkM16ttEHYvBUXbJP2mhHvpUCGRhcH5gufxaUY8L5PZkDboxxnNqZSzrE0VnSmUFi6d5vlGfWooZFouihnsoWQD6RQVu5V6MHEcS6XFSawD0LbxPFSW5CliAGsgGmPJljbZblR75U6yCFmVIbCmVZF8QJXLIMmwmZvJinKVpiqNNVGUPNKaT0nlWFJi+E1GqFRh5nqcROMpKQLdE8GIdaq1ZTl81itEADEWxXYG00pWl5qadNRg6XZfkGSh9Z0tQDJcjyfICnQ5lHPWeYFgldCTNCySeXpb1QNFTlfT9Szcry-KEHQwVDNwDYNnq3wALQWEdMT9d8Xn0mkv1owDWMk02OUAPJ2PAzDwjkT0Kj0hUWt4VpgAAvELi5-ejQzNROR0KhLqLtbkwYPHLXWASAJJAA).

**Comment by Timmmm** [comment]
Thanks! The overlapping turned out to be [a weird bug](https://github.com/microsoft/SandDance/issues/135) in SandDance's DeckGL backend for Vega, but documentation is always nice!
